graph/src/blockchain/firehose_block_stream.rs

```diff
@@ -125,20 +120,23 @@ where
     where
         F: FirehoseMapper<C> + 'static,
     {
+        if !client.is_firehose() {
+            panic!("Firehose block stream called with rpc endpoint");
```

Comment:
Is this user error, i.e., a misconfiguration or a code error? If it's user error would be good to tell them to change their config. Basically: if a user sees this error what should they do? Fix their config or report a bug?

Reply:
This is also a case of reporting a bug as there would have been some code path that called this when it was not using firehose

Reply:
Actually, if these are fatal code problems, then it's better to use `unreachable!` instead of `panic!`